doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -79,13 +89,15 @@ In deze tabel zijn de folders die door Action scripts worden gegenereerd niet op
 
 1. Maak de folders aan die voor de repository wenselijk zijn.
 
+VRAAG: Wanneer is een eigen template wenselijk? De templates zijn volgens mij aleen nodig wanneer je afwijkt van de standaard functionaliteit van github. 
```

Comment:
Misschien moeten we wel gewoon naar een aantal standaard templates die altijd aangeboden worden. Een template is een hulpmiddel dat je je gebruikers aanbiedt om het voor hen eenvoudiger te maken issues in te brengen. 
Zo kun je altijd een template aanbieden voor 'user stories', 'bugs', 'vragen', etc...

Reply:
Een template maakt het juist/ook makkelijker om voor een specifiek project tickets aan te maken.
Bijvoorbeeld door een issue altijd standaard aan een specifieke persoon toe te kennen, om het ticket te beoordelen. Een template is dan handig als basis maar voegt pas echt wat toe wanneer het voor het specifieke project aangepast is.

Reply:
Ik stel voor om een beperkte set van templates aan te maken die dan door de project maintainer zelf verbijzonderd kan worden. Daarmee creëren we eenheid over de repositories en een houvast voor de maintainer om op basis daarvan aanpassingen aan te brengen.
Laten we hier samen eens doorheen gaan en bepalen welke templates en hoe deze eruit moeten zien. Wellicht daarvoor ook een GitHub repo aanmaken dat als voorbeeld kan dienen voor de standaard inrichting. Of hebben we die misschien al?